File: youlexuan_manager_web/src/main/webapp/js/service/specificationService.js
app.service('specificationService',function ($http) {
    //列表 搜索功能
    this.seach=function (page, rows, SeachEntity) {
        return $http.post('../specification/seach.do?page='+page+'&rows='+rows,SeachEntity);
    }
    //增加
    this.save=function (entity) {
        return $http.post('../specification/add.do',entity);
    }
});

File: youlexuan_sellergoods_interface/src/main/java/com/offcn/sellergoods/service/SpecificationService.java
package com.offcn.sellergoods.service;

import com.offcn.entity.PageResult;
import com.offcn.group.Specification;
import com.offcn.pojo.TbSpecification;

public interface SpecificationService {

    public PageResult seach(TbSpecification specification,int pageNum,int pageSize);

    /**
     * 增加
     */
    public void add(Specification specification);
}
